components/studio/libexec/hab-studio-type-stage1.sh

```diff
@@ -12,23 +13,25 @@ studio_build_command=
 studio_run_environment=
 studio_run_command="/tools/bin/bash --login"
 
-: ${STAGE1_TOOLS_URL:=http://s3-us-west-2.amazonaws.com/habitat-studio-stage1/habitat-studio-stage1-20180312233639.tar.xz}
-: ${TAR_DIR:=/tmp}
```

Comment:
Unrelated to these lines specifically, but sometimes it appears that variables have defaults defined like so:

`${myvar:-loldefault}`

and sometimes like so:

`${myvar:=loldefault}`

What's the difference between the two?

Reply:
There's a great summary here: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02